utils/staking-miner/src/main.rs

```diff
@@ -630,8 +630,7 @@ async fn main() {
 			sp_core::crypto::set_default_ss58_version(
 				sp_core::crypto::Ss58AddressFormatRegistry::PolkadotAccount.into(),
 			);
-			sub_tokens::dynamic::set_name("WND");
-			sub_tokens::dynamic::set_decimal_points(1_000_000_000_000);
+			Token::init(WrappedToken { name: "WND", decimals: 12 });
```

Comment:
hmm westend is not part of this crate? strange..

Reply:
Yeah, it is missing from the JSON in the ss58-registry

Reply:
https://github.com/paritytech/ss58-registry/issues/85